docs/rule/no-obscure-hbs.md

```diff
@@ -0,0 +1,39 @@
+# no-obscure-hbs
+
+Using obscrure expressions `{{list.[1].name}}` is descouraged and might be deprecated soon
+
+This rule suggests usage of the Ember's getter instead.
```

Comment:
Should we really suggest Ember's `get` function? I thought Ember is moving away from using its custom `get` function:

https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-get.md

Reply:
I meant "get"  helper not the "get" function.  I'll update this description.